pkg/engine/engine.go

```diff
@@ -420,23 +426,16 @@ func (e *Engine) Logs(ctx context.Context, id string, follow bool, cancel bool,
 		}
 		defer file.Close()
 
-		scanner := bufio.NewScanner(file)
```

Comment:
@hacdias no need to read the file line by line (which already includes `ndjson` and encoding once again further down with `ow.WriteProgress`.